### PR TITLE
Observe Worker initialization

### PR DESCRIPTION
**Pull request by kevinfarrugia**
Required for [JavaScript 2022](https://github.com/HTTPArchive/almanac.httparchive.org/issues/2880)

Added support to count the number of times the Worker constructor is called. 

Test: https://webpagetest.httparchive.org/result/220529_6B_M/1/details/#waterfall_view_step1

Uses custom metric:
```
[observers]
return Object.fromEntries(Object.entries(httparchive_observers).filter(([_, value]) => value));
```

Results include:

observers | { Worker: 1 }
-- | --

### CODE REVIEW

**Comment by rviscomi** [comment]
I think all that's needed is a minimal change to add `Worker` to the `OBSERVERS` list. There was a bug in the implementation that I recently fixed in #32 where top-level properties under `window` weren't being observed properly. That might by why you thought you needed a separate implementation for constructors.

Here's a [test](https://webpagetest.httparchive.org/result/220528_MP_4/1/details/#waterfall_view_step1) where I just added the constructor name to the const, and it worked.

image

**Comment by kevinfarrugia** [comment]
Thank you @rviscomi . I have updated the PR and it seems to work correctly.

**Comment by rviscomi** [comment]
Did you push your changes? I'm still seeing the separate `CONSTRUCTORS` array.

**Comment by kevinfarrugia** [comment]
😴. Updated.

File: dist/javascript.js [hunk 1]
```diff
@@ -91,6 +91,9 @@ return JSON.stringify({
         };
 
         return noscript_data;
-    })()
+    })(),
 
+    workers: (() => {
```
Comment (rviscomi) [inline dist/javascript.js:96]: Similarly I think we can rely on the `observers.js` custom metric to extract this value. Otherwise LGTM.

**Review by rviscomi — CHANGES_REQUESTED** [review]

**Comment by kevinfarrugia** [comment]
Updated test using `observers.js`.

https://webpagetest.httparchive.org/result/220529_6B_M/1/details/#waterfall_view_step1

**Review by rviscomi — APPROVED** [review]

**Pull request merged**